clients/v1/clients.proto

```diff
@@ -39,6 +40,21 @@ service UsersService {
   rpc GetUserRoles(GetUserRolesRequest) returns (GetUserRolesResponse) {
     option (sams_required_scopes) = "sams::user.roles::read";
   };
+  // GetUserMetadata retrieves metadata for a SAMS user.
+  //
+  // Required scopes: 'sams::user.metadata::write' or metadata-scope-specific
+  // variant scope, such as 'sams::user.metadata.dotcom::write'
+  rpc GetUserMetadata(GetUserMetadataRequest) returns (GetUserMetadataResponse) {
+    // Cannot use 'sams_required_scopes' due to metadata-scope-specific scopes.
+    // See docstring for required scopes.
+  }
+  // UpdateUserMetadata updates the metadata of a SAMS user.
+  //
+  // Required scopes: 'sams::user.metadata::write' or metadata-scope-specific
+  // variant scope, such as 'sams::user.metadata.dotcom::write'
+  rpc UpdateUserMetadata(UpdateUserMetadataRequest) returns (UpdateUserMetadataResponse) {
+    // Cannot use 'sams_required_scopes' due to metadata-scope-specific scopes.
+  }
```

Comment:
@unknwon per AIP this would always `overwrite = true`, WDYT?

Reply:
yeah, this parameter:

![image](https://github.com/user-attachments/assets/200dc50d-ad36-422e-ba41-41a2831da83f)

https://sourcegraph.sourcegraph.com/github.com/sourcegraph/sourcegraph-accounts/-/blob/backend/internal/database/user_metadata.go?L47

Reply:
Yep, LGTM!